janusgraph-inmemory/src/test/java/org/janusgraph/graphdb/inmemory/InMemoryGraphTest.java

```diff
@@ -57,6 +59,11 @@ public void clopen(Object... settings) {
     @Override @Test @Disabled
     public void testLocalGraphConfiguration() {}
 
+    @Override @Test @Disabled
+    public void testLimitBatchSizeForMultiQuery() {
+        assertTrue(true);
```

Comment:
Rather than "assertTrue(true)", you could just put a comment here saying that this test is skipped, e.g. "//Do nothing".

Reply:
I can try, but I suspect that would still fail the codacy test that says "each test should call `assert` or `fail` at least once.

Reply:
I vaguely remember I encountered the same problem once and adding a dummy comment worked.

Reply:
As expected, the codacy check still fails with `{ /* do nothing */ }`.

Reply:
My memory was wrong then :(